### ISSUE TITLE
Narrator: Control type for 'More tab item' button is defined as menu item.

### ISSUE THREAD

**Issue by sanket856** [microsoft/vscode-docs#4975]
GitHubTags:#A11ySev2;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#Win10;#screenreader;#Narrator;#WCAG4.2.1:#VisualStudioCode-Win32-May2021

### Environment Details:​
Visual Studio Code.
Version: 1.62.3.

### Pre-Requisites:

1. 'Install 'Visual Studio Code' application.
2. Go to 'Extensions' and add 'Microsoft Edge(Chromium) Tools' extension.

### Repro Steps:​

1. Launch VS code application
2. Navigate to 'Microsoft Edge(Chromium) Tools' tab item and hit enter key.
3. Navigate to 'Open a New Tab' button and hit enter key.
5. Now navigate back to visual studio code application using ALT+TAB key or minimize the browser.
6. Navigate to 'More tab item' in style panel and listen to narrator's announcement.

### Actual Result:
Control type for 'More tab item' button is defined as menu item.

**NVDA Behavior:** This issue is not observed with NVDA as NVDA announces "More Tabs Menu button submenu".

### Expected Result:
Control type for 'More tab item' should be defined as button. And narrator should announce status as 'Expanded/Collapsed' upon invoking 'more tab item'.

### User Impact:
People who rely on screen reader will face difficulty to understand the functionality of 'More tab item' if control type is defined as menu item.

**Comment by gregvanl** [comment]
Hi @sanket856, VS Code desktop A11y issues should go to the vscode repo not the vscode-docs repo.
vscode-docs is for the website https://code.visualstudio.com documentation issues.
